sigilrecorder.lic

```diff
@@ -0,0 +1,205 @@
+custom_require.call(%w[common common-arcana common-items common-travel drinfomon events spellmonitor equipmanager])
+
+class SigilRecorder
+  include DRC
+  include DRCA
+  include DRCI
+  include DRCT
+
+	def initialize
+		require 'yaml'
+		@data = YAML.load(File.read("./scripts/data/base-sigils.yaml"))
+		@city_list = ['shard', 'crossing', 'riverhaven', 'other']
+		@sigil_list = ['abolition','congruence','induction','permutation','rarefaction', 'antipode','ascension','clarification','decay','evolution','integration','metamorphosis','nurture','paradox','unity']
```

Comment:
We already have this list defined in https://github.com/rpherbig/dr-scripts/blob/6d087f0ebb4dcf449ab854ace7afe0b5f0d884b0/attunement.lic#L14 - is there a way to avoid duplication?

Reply:
I am not making a distinction between primary or secondary sigils because when you need a sigil you don't really care which it is and you still have to spam 'perc sigil' until the one you want appears. I could generate the list used by my script by adding the two constants together. That way if new sigils are added in the future my script should automatically start saving data for the new sigils.

Reply:
Looking at Attunement I am also not sure how to convert those regex patterns to arrays. Arrays are what I am working with. Ideally it would be nice to automagically update my script if new sigils are added, though.

Reply:
Figured out my synax issues and am now using get_dat